app/src/main/res/layout/fragment_pass_recovery.xml

```diff
@@ -95,7 +95,7 @@
             app:layout_constraintVertical_bias="0.12">
 
             <TextView
-                android:onClick="@{(v) -> PasswordRecoveryViewModel.onResetClicked(v)}"
+                android:onClick="@{() -> PasswordRecoveryViewModel.onResetClicked()}"
```

Comment:
W sumie nie wiem dlaczego, ale wydawało mi się, że to (v) od View jest potrzebne, skoro nie to w sumie można usunąć też w pozostałych